code/game/machinery/vending.dm

```diff
@@ -1589,5 +1601,241 @@
 	if(choice)
 		SetName(choice)
 
-
+/obj/machinery/vending/Merc_clothing_vendor
+	name = "Merc'ury Clothing Solutions"
+	desc = "Your one Stop Shop for Surplus Military Equipment"
+	product_slogans = "Go gettem!"
+	product_ads = "YOU WANNA LIVE FOREVER!; One hundred percent Kevlar!; Not rated for giant Mantas Invasions.; Market research shows that flak vests will replace shirts entirely in a few years. Get ahead of the fashion!"
```

Comment:
This isn't your fault but god damn do I hate how the product ads are stored in vendor code!

Reply:
Same